src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -178,7 +178,54 @@ <h5><span jhiTranslate="artemisApp.studentExams.summary">Student Summary</span><
         </a>
     </div>
 
-    <div class="mb-3">
+    <div class="mb-3 mt-3">
```

Comment:
As @danielbode mentioned this does not seem to work well for smaller devices.
How about allowing that div to scroll horizontally?